### PR TITLE
Improvements in Site : Check Comments

### PR DESCRIPTION
**Pull request by RamGoel**
1- Made About Page (https://github.com/ElixirTechCommunity/Nexus_Website/issues/29)
4- CTA button Updated (https://github.com/ElixirTechCommunity/Nexus_Website/issues/26)

### CODE REVIEW

**Comment by pulkit-30** [comment]
Dev Preview : https://deploy-preview-36--hacknexus.netlify.app/dev.html

File: scripts/vanilla-tilt.js [hunk 1]
```diff
@@ -0,0 +1,511 @@
+var VanillaTilt = (function () {
```
Comment (pulkit-30) [inline scripts/vanilla-tilt.js:1]: This effect is done in #34

**Review by pulkit-30 — CHANGES_REQUESTED** [review]

File: scripts/vanilla-tilt.js (hunk 1 above)
Comment (Anv3sh) [reply scripts/vanilla-tilt.js:1]: @RamGoel  can you please revert this commit else we will face a merge conflict.

File: dev.html [hunk 2]
```diff
@@ -1,106 +1,107 @@
...
+    <nav class="navbar flex">
+      <a class="flex" href="#home">
+        NE
+        <img src="./assets/favicon.svg" width="60" alt="X" />
+        US
+      </a>
+      <div class="flex">
+        <a class="headLinks" href="https://twitter.com/nexus_hackathon"><i
+            class=" linkIcon fa-brands fa-twitter"></i></a>
+        <a class="headLinks" href="https://www.instagram.com/nexus_whatthehack/"><i
```
Comment (Anv3sh) [inline dev.html:39]: You don't need to change the structure of the codebase just add the links in the `href` . The class names are defined for a reason don't change them for your convenience.

**Review by Anv3sh — CHANGES_REQUESTED** [review]

**Comment by RamGoel** [comment]
Made Required Changes

**Comment by RamGoel** [comment]
@pulkit-30 @Anv3sh Check Now

**Review by dewank07 — CHANGES_REQUESTED** [review]
also revert back the changes done in stylesheets/slide_page.css for glassmorphism they are already did in #34

**Comment by Anv3sh** [comment]
@RamGoel please resolve the merge conflict.

**Comment by RamGoel** [comment]
@Anv3sh @pulkit-30 Check Now

**Pull request closed without merging**